feature/signupflow/entercode/src/main/java/com/epmedu/animeal/signup/entercode/presentation/ui/CodeRow.kt

```diff
@@ -50,14 +50,14 @@ private fun CodeRowPreview() {
     AnimealTheme {
         Column {
             CodeRow(
-                code = listOf(1, 2, null, null),
+                code = listOf(1, 2, null, null, null, null),
```

Comment:
Can you also replace list with nulls in EnterCodeScreenPreview with emptyCode() call from companion in viewmodel?